pkg/cmd/admin/network/project_options.go

```diff
@@ -84,7 +84,14 @@ func (p *ProjectOptions) Validate() error {
 		errList = append(errList, errors.New("must provide --selector=<project_selector> or projects"))
 	}
 
-	// TODO: Validate if the openshift master is running with mutitenant network plugin
```

Comment:
When we added this CLI functionality, there is no way to fetch network plugin info from the master. Now, ClusterNetwork resource holds the plugin name which enables this validation. 
@xiangpengzhao Thanks for your contribution.